modules/ipi-install-network-requirements.adoc

```diff
@@ -15,13 +15,13 @@ Installer-provisioned installation of {product-title} involves several network r
 +
 In {product-title} 4.3, when deploying using the `provisioning` network, the first NIC on each node, such as `eth0` or `eno1`, must interface with the `provisioning` network.
 +
-In {product-title} 4.4 and later releases, you can specify the provisioning network NIC with the `provisioningNetworkInterface` configuration setting.
+In {product-title} 4.4 and later releases, the `bootMacAddress` field is used to calculate the IP address for Ironic to bind to. The `provisioningNetworkInterface` configuration field is optional.
```

Comment:
Maybe:
...configuration field is now optional.

Reply:
Updated as suggested.